reference_data/management/commands/update_dbnsfp_gene.py

```diff
@@ -0,0 +1,261 @@
+                "pathway_kegg_id": fields['Pathway(KEGG)_id'],           # ID(s) of the Pathway(s) the gene belongs to (from KEGG)
+                "pathway_kegg_full": fields['Pathway(KEGG)_full'],         # Full name(s) of the Pathway(s) the gene belongs to (from KEGG)
+                "function_desc": fields['Function_description'].replace("FUNCTION: ", ""),  # Function description of the gene (from Uniprot)
+                "disease_desc": fields['Disease_description'].replace("FUNCTION: ", ""),    # Disease(s) the gene caused or associated with (from Uniprot)
+                "trait_association_gwas": fields['Trait_association(GWAS)'], # Trait(s) the gene associated with (from GWAS catalog)
+                "go_biological_process": fields['GO_biological_process'],   # GO terms for biological process
+                "go_cellular_component": fields['GO_cellular_component'],   # GO terms for cellular component
+                "go_molecular_function": fields['GO_molecular_function'],   # GO terms for molecular function
+                "tissue_specificity": fields['Tissue_specificity(Uniprot)'],   # Tissue specificity description from Uniprot
+                "expression_egenetics": fields['Expression(egenetics)'],   # Tissues/organs the gene expressed in (egenetics data from BioMart)
+                "expression_gnf_atlas": fields['Expression(GNF/Atlas)'],   # Tissues/organs the gene expressed in (GNF/Atlas data from BioMart)
+                "rvis_exac": fields['RVIS_ExAC'],
+                "ghis": fields['GHIS'],
+                "essential_gene": fields['Essential_gene'],   # Essential ("E") or Non-essential phenotype-changing ("N") based on Mouse Genome Informatics database. from doi:10.1371/journal.pgen.1003484
+                "mgi_mouse_gene": fields['MGI_mouse_gene'],   # Homolog mouse gene name from MGI
+                "mgi_mouse_phenotype": fields['MGI_mouse_phenotype'],   # Phenotype description for the homolog mouse gene from MGI
+                "zebrafish_gene": fields['ZFIN_zebrafish_gene'],   # Homolog zebrafish gene name from ZFIN
+                "zebrafish_structure": fields['ZFIN_zebrafish_structure'],   # Affected structure of the homolog zebrafish gene from ZFIN
+                "zebrafish_phenotype_quality": fields['ZFIN_zebrafish_phenotype_quality'],   # Phenotype description for the homolog zebrafish gene from ZFIN
+                "zebrafish_phenotype_tag": fields['ZFIN_zebrafish_phenotype_tag'],   # Phenotype tag for the homolog zebrafish gene from ZFIN
+            })
+
+    print("Parsed {} records. Inserting them into dbNSFPGene".format(len(records)))
+
+    dbNSFPGene.objects.bulk_create((dbNSFPGene(**record) for record in tqdm(records, unit=' genes')), batch_size=1000)
+
+    logger.info("Done loading {} records into dbNSFPGene".format(dbNSFPGene.objects.count()))
+
+
+"""
+Columns of dbNSFP_gene:  (from dbNSFP REAMDE v3.5a)
```

Comment:
I generally dislike keeping the model schemas in comments like this because they get so quickly out of date

Reply:
dbNSFP usually takes 1+ years between updates and the README is hard to find. I left this in case users ask for fields that are in dbNSFP - so it's easy to look up.

Reply:
that makes sense, but couldn't you just leave a link to the readme?

Reply:
good point.. will do that